fxa-oauth-server/docs/api.md

```diff
@@ -377,33 +379,45 @@ If requesting an implicit grant (token), the response will match the
 
 ### POST /v1/token
 
-After having received a [code][authorization], the client sends that code (most
-likely a server-side request) to this endpoint, to receive a
-longer-lived token that can be used to access attached services for a
-particular user.
+After receiving an authorization grant from the user, clients exercise that grant
+at this endpoint to obtain tokens that can be used to access attached services
+for a particular user.
 
-#### Request Parameters
+The following types of grant are possible:
+- `authorization_code`: a single-use code as produced by the [authorization][] endpoint,
+  obtained through a redirect-based authorization flow.
+- `refresh_token`: a token previously obtained from this endpoint when using
+  `access_type=offline`.
+- `fxa-assertion`: an FxA identity assertion, obtained by directly authenticating
+  the user's account.
 
-- `ttl`: (optional) Seconds that this access_token should be valid.
+#### Request Parameters
 
+- `ttl`: (optional) Seconds that the access_token should be valid.
   The default and maximum value is 2 weeks.
```

Comment:
Default will be 12 days as soon as https://github.com/mozilla-services/cloudops-deployment/pull/3028 is deployed.